client/extensions/woocommerce/app/settings/email/mailchimp/setup-steps/newsletter-settings.js

```diff
@@ -0,0 +1,61 @@
+/**
+ * External dependencies
+ */
+import { connect } from 'react-redux';
+import PropTypes from 'prop-types';
+import React from 'react';
+import { localize } from 'i18n-calypso';
+import { map } from 'lodash';
+
+/**
+ * Internal dependencies
+ */
+import { getSelectedSiteId } from 'state/ui/selectors';
+import { isRequestingLists } from 'woocommerce/state/sites/settings/email/selectors';
+import FormFieldset from 'components/forms/form-fieldset';
+import FormLabel from 'components/forms/form-label';
+import FormSelect from 'components/forms/form-select';
+import QueryMailChimpLists from 'woocommerce/state/sites/settings/email/queryLists';
+
+const NewsletterSettings = ( { storeData, onChange, siteId, isRequesting, translate } ) => {
```

Comment:
Same suggestion on `storeData` here, add a default `{}` and/or make isRequired in the prop types.